src/Sentry/SentryId.cs

```diff
@@ -14,12 +15,16 @@ namespace Sentry
         /// <summary>
         /// An empty sentry id.
         /// </summary>
-        public static readonly SentryId Empty = Guid.Empty;
+        public static readonly SentryId Empty = new();
 
         /// <summary>
         /// Creates a new instance of a Sentry Id.
         /// </summary>
-        public SentryId(Guid guid) => _guid = guid;
+        public SentryId(Guid guid)
+        {
+            Debug.Assert(guid != Guid.Empty, "Dont use this API with Guid.Empty. Instead use SentryId.Empty");
```

Comment:
Since this is a struct, the result would be the same.  `SentryId.Empty == new SentryId(Guid.Empty)` via value equality, not reference equality.